### DISCUSSION TITLE
Happy to contribute

### DISCUSSION THREAD

**Discussion by TrendAndrew** [trendmicro/solutions-architect#51] [General]
Programming veteran of 40 years, happy to put my hand up for a few jobs to help out.

I'll turn on the watch and listen to activity here.

Already built some Node-red nodes to surface cloud one APIs to the node-red flows:
https://github.com/TrendAndrew/node-red-conformity
https://github.com/TrendAndrew/node-red-workloadsecurity

and built a cloud inventory tool to use the conformity APIs to extract reports on resources in accounts:
https://github.com/TrendAndrew/cloudInventory

**Comment by felipecosta09** [comment]
Welcome, Andrew!!

We appreciate it the help a lot, feel free to comment on any PR's and help us with your code experience, we need more code reviewers...

I love the project! I can't wait to see the PR's 😄
